abdiff/core/utils.py

```diff
@@ -11,26 +11,40 @@
 
 def read_job_json(job_directory: str) -> dict:
     """Read job JSON file."""
-    job_json_filepath = Path(job_directory) / "job.json"
-    with open(job_json_filepath) as f:
+    with open(Path(job_directory) / "job.json") as f:
         return json.load(f)
 
 
-def update_or_create_job_json(job_directory: str | Path, new_job_data: dict) -> dict:
-    """Create or update a job's JSON file.
+def read_run_json(run_directory: str) -> dict:
+    """Read run JSON file."""
+    with open(Path(run_directory) / "run.json") as f:
+        return json.load(f)
+
+
+def update_or_create_json(
+    directory: str | Path,
```

Comment:
Good call on the more generic function, same `Path` question as the previous iteration of the function